openapi/pacta.yaml

```diff
@@ -656,19 +534,18 @@ components:
           type: boolean
           description: Whether the given user is a super admin
 
-    EmptySuccess:
-      type: object
-
     Error:
       type: object
       required:
-        - code
         - message
+        - error_id
       properties:
-        code:
-          type: integer
-          format: int32
-          description: Error code
         message:
           type: string
-          description: Error message
+          description: Human readable error message (in English)
+        error_id:
+          type: string
+          description: |
+            An enum-like type indicating a more specific type of error.
+
+            TODO: Add an example
```

Comment:
If you're revisiting/visiting this question for an example, I think the error for authn for 2+ emails on the token is an excellent candidate.